NEWS.md

```diff
@@ -1,5 +1,10 @@
 # MpiIsoApp development version
 
+## MpiIsoApp 23.01.5
```

Comment:
Can you use 23.01.4 here? Even if there is another PR in the pipeline: "wer zuerst merged, mahlt zuerst..." ;)

Regarding the other PR, I first want to ensure with Ricardo that the name for the new Repo is fine for him. And only after that, I will merge ...